apps/prairielearn/src/pages/administratorInstitutions/administratorInstitutions.html.ts

```diff
@@ -89,3 +113,266 @@ export function AdministratorInstitutions({
+                <label for="short_name">Short name</label>
+                <input
+                  type="text"
+                  class="form-control"
+                  id="short_name"
+                  name="short_name"
+                  placeholder="Short name"
+                />
+              </div>
+              <div class="form-group">
+                <label for="long_name">Long name</label>
+                <input
+                  type="text"
+                  class="form-control"
+                  id="long_name"
+                  name="long_name"
+                  placeholder="Long name"
+                />
+              </div>
+              <div class="form-group">
+                <label for="display_timezone">Timezone</label>
+                <input
+                  type="text"
+                  class="form-control"
+                  id="display_timezone"
+                  name="display_timezone"
+                  placeholder="Timezone"
+                />
+              </div>
+              <div class="form-group">
+                <label for="uid_regexp">UID regexp</label>
+                <input
+                  type="text"
+                  class="form-control"
+                  id="uid_regexp"
+                  name="uid_regexp"
+                  placeholder="UID regexp"
+                />
+              </div>
+              <div class="form-group">
+                <label for="authn_providers"
+                  >Authn providers
+                  <p>
+                    <small
+                      >Note: authn providers must be separated by a comma and a space (i.e, "Azure,
+                      Google")
+                    </small>
+                  </p>
+                  <p></p
+                ></label>
+                <input
+                  type="text"
+                  class="form-control"
+                  id="authn_providers"
+                  name="authn_providers"
+                  placeholder="Authn providers"
+                />
+              </div>
+              <div class="form-group">
+                <label for="course_instance_enrollment_limit"
+                  >Course instance enrollment limit</label
+                >
+                <input
+                  type="text"
+                  class="form-control"
+                  id="course_instance_enrollment_limit"
+                  name="course_instance_enrollment_limit"
+                  placeholder="Course instance enrollment limit"
+                />
+              </div>
+              <div class="form-group">
+                <label for="yearly_enrollment_limit">Yearly enrollment limit</label>
+                <input
+                  type="text"
+                  class="form-control"
+                  id="yearly_enrollment_limit"
+                  name="yearly_enrollment_limit"
+                  placeholder="Yearly enrollment limit"
+                />
+              </div>
```

Comment:
I think we can remove these from the creation page; these can be set immediately after creation.

Reply:
I have removed these sections from the creation modal in commit [402fad6](https://github.com/PrairieLearn/PrairieLearn/pull/9045/commits/402fad6c5a45a0e8ff5187613d3563a1bbf7d8f0).